[2021-12-13 15:24:50,783] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: cleansing_dag.writeOut 2021-12-12T15:24:50.188872+00:00 [queued]>
[2021-12-13 15:24:50,804] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: cleansing_dag.writeOut 2021-12-12T15:24:50.188872+00:00 [queued]>
[2021-12-13 15:24:50,805] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-12-13 15:24:50,805] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2021-12-13 15:24:50,805] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-12-13 15:24:50,829] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): writeOut> on 2021-12-12T15:24:50.188872+00:00
[2021-12-13 15:24:50,833] {standard_task_runner.py:52} INFO - Started process 7104 to run task
[2021-12-13 15:24:50,837] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'cleansing_dag', 'writeOut', '2021-12-12T15:24:50.188872+00:00', '--job-id', '8', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/kym_cleansing.py', '--cfg-path', '/tmp/tmp78xd7prr', '--error-file', '/tmp/tmp2_t0o9a6']
[2021-12-13 15:24:50,837] {standard_task_runner.py:77} INFO - Job 8: Subtask writeOut
[2021-12-13 15:24:50,875] {logging_mixin.py:104} INFO - Running <TaskInstance: cleansing_dag.writeOut 2021-12-12T15:24:50.188872+00:00 [running]> on host e888e487975c
[2021-12-13 15:24:50,914] {taskinstance.py:1255} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=cleansing_dag
AIRFLOW_CTX_TASK_ID=writeOut
AIRFLOW_CTX_EXECUTION_DATE=2021-12-12T15:24:50.188872+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-12T15:24:50.188872+00:00
[2021-12-13 15:24:51,057] {python.py:118} INFO - Done. Returned value was: None
[2021-12-13 15:24:51,080] {taskinstance.py:1159} INFO - Marking task as SUCCESS. dag_id=cleansing_dag, task_id=writeOut, execution_date=20211212T152450, start_date=20211213T152450, end_date=20211213T152451
[2021-12-13 15:24:51,130] {local_task_job.py:146} INFO - Task exited with return code 0
